tox.ini

```diff
@@ -28,11 +28,6 @@ envlist =
     # aiobotocore 0.2 and 0.4 do not work because they use async as a reserved keyword
     aiobotocore_contrib-py{37,38}-aiobotocore{03,05,07,08,09,010,011,012}
     aiobotocore_contrib-py39-aiobotocore{010,011,012}
-    # Python 3.7 needs at least aiohttp 2.3
-    aiohttp_contrib-{py35,py36}-aiohttp{20,21,22}-aiohttp_jinja{012,013}-yarl-pytest3
```

Comment:
you can delete the listed dependencies (`aiohttp20`, `aiohttp21`, etc) below as well